packages/mdc-floating-label/mdc-floating-label.scss

```diff
@@ -26,7 +27,7 @@
   bottom: 8px;
   left: 0;
   transform-origin: left top;
-  transition: mdc-text-field-transition(transform), mdc-text-field-transition(color);
+  transition: mdc-floating-label-transition(transform), mdc-floating-label-transition(color);
   cursor: text;
 
   // stylelint-disable plugin/selector-bem-pattern
```

Comment:
Should the block below this comment (`@include mdc-rtl(".mdc-text-field")`) get moved to text-field?

Reply:
Keeping it, but removing mdc-text-field. Thanks for catching